app/admin/solicitation.rb

```diff
@@ -33,6 +33,17 @@
         end
       end
       blockquote simple_format(s.description&.truncate(20000, separator: ' '))
+
+      ul class: 'mb-1' do
+        s.institution_filters.each do |filter|
+          if t(filter.key, scope: [:activerecord, :attributes, :additional_subject_questions]).has_key?(:true)
```

Comment:
Je me demande si avec un `default`, ça marcherait pas ? 
https://coderwall.com/p/1joa7g/scoping-and-default-for-i18n-rails

(si oui, il me semble qu'il y a le même code qqpart ailleurs. Etape vérification ?)